extensions/azurecore/package.json

```diff
@@ -140,7 +140,7 @@
 				},
 				{
 					"command": "azureresource.refresh",
-					"when": "viewItem != azureResource.itemType.database && viewItem != azureResource.itemType.databaseServer && viewItem != azureResource.itemType.message",
+					"when": "viewItem =~ /^azureResource.itemType./ && viewItem != azureResource.itemType.database && viewItem != azureResource.itemType.databaseServer && viewItem != azureResource.itemType.message",
```

Comment:
2 questions:
- Doesn't the `.` match any character, per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp ? Should you be escaping it when used in regex?
- What items is this expected to match? Thanks.